packages/channel-client/src/channel-client.ts

```diff
@@ -105,11 +105,11 @@ export class ChannelClient implements ChannelClientInterface<ChannelResult> {
     });
   }
 
-  async getBudget(hubAddress: string): Promise<SiteBudget> {
+  async getBudget(hubAddress: string): Promise<SiteBudget | {}> {
```

Comment:
Why `| {}`?

Reply:
We need some return type when `getBudget` is called and no budget exists, I wasn't sure the best way to represent this over JSON-RPC but I used `{}` for now.

Another option would be to return an empty `SiteBudget` but I think that may be less clear? (Or maybe just a `null` result`)?